hydra-cluster/test/Test/CardanoNodeSpec.hs

```diff
@@ -39,11 +38,13 @@ spec = do
         slot2 <- queryTipSlotNo networkId nodeSocket
         slot2 `shouldSatisfy` (> slot1)
 
-    it "withCardanoNodeOnKnownNetwork on mainnet starts synchronizing within 5 seconds" $ \(tr, tmp) ->
+    it "withCardanoNodeOnKnownNetwork on mainnet starts synchronizing within 5 seconds" $ \(_, _) -> pendingWith "not yet supported"
```

Comment:
Could: be more specific on the pending reason

For example mentioning that the genesis conway files downloaded by `withCardanoNodeOnKnownNetwork` are not yet compatible with cardano-node 8.8

Reply:
Done in https://github.com/input-output-hk/hydra/commit/55d55dbdd031639dfa19cc57d4e0c1eeb1af2b8d